src/uct/ib/mlx5/dv/ib_mlx5dv_md.c

```diff
@@ -546,6 +546,28 @@ uct_ib_mlx5_devx_check_odp(uct_ib_mlx5_md_t *md,
     return UCS_OK;
 }
 
+static ucs_status_t
+uct_ib_mlx5_devx_query_lag(uct_ib_mlx5_md_t *md, uint8_t *state)
+{
+    char out[UCT_IB_MLX5DV_ST_SZ_BYTES(query_lag_out)] = {};
+    char in[UCT_IB_MLX5DV_ST_SZ_BYTES(query_lag_out)]   = {};
+    void *lag;
+    int ret;
+
+    lag = UCT_IB_MLX5DV_ADDR_OF(query_lag_out, out, lag_context);
+    UCT_IB_MLX5DV_SET(query_lag_in, in, opcode, UCT_IB_MLX5_CMD_OP_QUERY_LAG);
+    ret = mlx5dv_devx_general_cmd(md->super.dev.ibv_context, in, sizeof(in),
+                                  out, sizeof(out));
+    if (ret != 0) {
+        ucs_error("mlx5dv_devx_general_cmd(QUERY_LAG) failed: %m");
+        return UCS_ERR_IO_ERROR;
+    }
+
+    *state = UCT_IB_MLX5DV_GET(lag_context, lag, lag_state);
+
```

Comment:
remove space line

Reply:
@yosefe Fixed.

Reply:
pls remove space line from line 566 and add space line between  564 and 565